### ISSUE TITLE
Make an option to ignore certain holidays in get_working_days_delta.

### ISSUE THREAD

**Issue by lucafs** [workalendar/workalendar#689]
# Feature
get_working_days_delta should have a parameter called ignore_holidays that should be a list of strings with the name of the holidays that you want to ignore in the function.

```
cal = France()
cal.get_working_days_delta(inicial_date, final_date, ignore_holidays  = ['New year' , 'Labour Day']) 

```

**Comment by ewjoachim** [comment]
Such a feature where you select names by their "human readable" names will make it extremely complex to work when we implement localization. Also, it means that if someone fixes a typo in an human readable name, it may break someone else's code. I think the proposal is interesting, but we should try and find a better approach.

**Comment by brunobord** [comment]
I agree with @ewjoachim here. Holidays label are likely to change. The `get_working_days_delta` method already has `extra_working_days` and `extra_holidays` arguments to exclude/include days in your computation. So you already have the ability to chose arbitrary days to ignore or not, based on your specifics.

And still, if you have *very specific* needs about holidays you'd like to skip, you may create your own class, based on an existing "`France`" class or not, and "make your own mix" of holidays.

Is it okay for you?
